### PR TITLE
Create pull request template

### PR DESCRIPTION
**Pull request by FionnD**
Pull request template for all future PRs. To ensure we don't forget anything.

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+### All Submissions:
+
+* [ ] Have you followed the documentation style guide?
```
Comment (nan-wang) [inline .github/pull_request_template.md]: Do we need to add the URL link here?

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+### All Submissions:
+
+* [ ] Have you followed the documentation style guide? 
+
+<!-- You can erase any parts of this template not applicable to your Pull Request. -->
+
+### New Feature Submissions:
+
+1. [ ] Have you added your new file to a table of content index? 
+2. [ ] Did you use one of the templates in the page_template folder?
```
Comment (nan-wang) [inline .github/pull_request_template.md]: The same question goes here.

**Review by nan-wang — CHANGES_REQUESTED** [review]

File: .github/pull_request_template.md (hunk 1 above)
Comment (FionnD) [reply .github/pull_request_template.md]: Done!

File: .github/pull_request_template.md (hunk 2 above)
Comment (FionnD) [reply .github/pull_request_template.md]: Done!

**Review by nan-wang — APPROVED** [review]

**Pull request merged**